src/FeedsCleanup/Program.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+using NuGet;
+
+namespace FeedsCleanup
+{
+    /// <summary>
+    /// Removing unused packages from both local CI feed and volatile feed
+    /// </summary>
+    class Program
+    {
+        public static int Main(string[] args)
+        {
+            var app = new CommandLineApplication(true);
+            app.Command("local", CleanLocal);
+            app.Command("remote", CleanRemote);
+
+            app.HelpOption("-?|-h|--help");
+
+            return app.Execute(args);
+        }
+
+        private static void CleanLocal(CommandLineApplication context)
+        {
+            context.Description = "Clean up the local latest-packages folder";
+            var argCoherenceFolder = context.Argument("coherent",
+                "The folder contains the coherent packages", multipleValues: false);
+            var argLatestFolder = context.Argument("latest-ci",
+                "The local latest pacakges folder on CI", multipleValues: false);
+            var optWhatif = context.Option("-w|--whatif",
+                "Print the actions will be taken instead of actually take them", CommandOptionType.NoValue);
```

Comment:
Change message to: `Instructs the command to simulate the actions that it would take without applying any changes.`